### PR TITLE
feat(helm:create): allow absolute paths when specifying starter template path [master]

### PR DESCRIPTION
**Pull request by mentos1386**
**What this PR does / why we need it**:

If starter template path is an absolute path, it shouldn't be prefixed with c.home.Starters() but rather be used as is. This way we can use templates that might not be in home directory but inside a specific project.

PR to `dev-v3` branch is #5985  
**Special notes for your reviewer**:

**If applicable**:
- [x] this PR contains documentation
- [x] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

**Comment by bacongobbler** [comment]
thanks for the PR! Would you be willing to port this over to the `dev-v3` branch as well? That way it'll make its way into Helm 3. Thanks!

**Comment by bacongobbler** [comment]
Also would you mind writing a unit test for this?

**Comment by mentos1386** [comment]
@bacongobbler should i close this PR and create a new one for `dev-v3`? Or create another one to `dev-v3`?

**Comment by bacongobbler** [comment]
Create a similar one for dev-v3

**Comment by mentos1386** [comment]
@bacongobbler Hey, just want to inform you that this is done from my side. Also the dev-v3 version

File: cmd/helm/create.go [hunk 1]
```diff
@@ -106,6 +107,10 @@ func (c *createCmd) run() error {
 	if c.starter != "" {
 		// Create from the starter
 		lstarter := filepath.Join(c.home.Starters(), c.starter)
+		// If path is absolute, we dont want to prefix it with helm starters folder
+		if strings.HasPrefix(c.starter, "/") {
```
Comment (bacongobbler) [inline cmd/helm/create.go]: Can you use `filepath.IsAbs` instead? https://golang.org/pkg/path/filepath/#IsAbs

**Review by bacongobbler — CHANGES_REQUESTED** [review]

File: cmd/helm/create.go (hunk 1 above)
Comment (mentos1386) [reply cmd/helm/create.go]: Fixed, also in v3 pr

**Review by bacongobbler — APPROVED** [review]

**Comment by mentos1386** [comment]
@bacongobbler @mattfarina PR for v3 https://github.com/helm/helm/pull/5985 was merged last week. Shouldn't this be also merged?

**Comment by bacongobbler** [comment]
PRs require 2 LGTMs to be merged. So far there's just one. :)

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @mentos1386

**Pull request merged**